boards/nxp/frdm_mcxa166/board.c

```diff
@@ -15,12 +15,23 @@
 /* System clock frequency. */
 extern uint32_t SystemCoreClock;
 
+static void enable_lpcac(void)
+{
+	SYSCON->LPCAC_CTRL |= SYSCON_LPCAC_CTRL_LPCAC_MEM_REQ_MASK;
+	SYSCON->LPCAC_CTRL |= SYSCON_LPCAC_CTRL_CLR_LPCAC_MASK;
+	SYSCON->LPCAC_CTRL &=
+		~(SYSCON_LPCAC_CTRL_CLR_LPCAC_MASK | SYSCON_LPCAC_CTRL_DIS_LPCAC_MASK);
+}
+
```

Comment:
This is already done in the HAL as part of `SystemInit`. I don't think this is needed.